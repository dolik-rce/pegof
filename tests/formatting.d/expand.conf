input formatting.d/expand.peg
