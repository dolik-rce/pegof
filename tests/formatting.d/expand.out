X <- <"A"> <.> <"C"> $1 $2 $1 $3
